docs/custom_mixer.md

```diff
@@ -0,0 +1,41 @@
+# Deploy Custom Mixer
+
+Mixer can be hosted in your own GCP project to serve SPARQL query for your data.
+
+This tutorial lists the steps to set this up.
+
+(TODO: Add more details to the following steps.)
+
+## Create Project
+
+Create a new GCP project and setup billing account.
+
+## Import Dataset
+
+* Create Bigquery Dataset and import tables from csv files.
+* Create template MCF files (schema mapping) based on the Bigquery dataset.
+
+## Setup GKE
+
+* Follow the [instructions](../gke/README.md) to setup the GKE instance.
+
+## Deployment
+
+* Remove all the .mcf files from [mapping](../deploy/mapping) directory and add
+  the custom template MCF files created above in that directory.
+
+* Change the mapping customization [file](../deploy/mapping/kustomization.yaml)
+  to use the custom MCF files.
+
+* Update the bigquery version [file](../deploy/storage/bigquery.version) to use
+  the custom dataset.
+
```

Comment:
So, typically would people start by "forking" the mixer over and then making these changes?

And how will they patch new updates/bug-fixes to it?

Reply:
These are config changes to the particular project. If people would contribute to the common code part, should be able to follow the regular git PR process.

Reply:
I understand this is specific to a particular project. My question was how those people will track the configs? If they create a fork of our repo, and make local changes, then keeping it up to date with our fixes is a pain. What's our recommendation?

Reply:
Normally forked repo owner would do "git pull dc master --rebase" to pull from the base repo. I moved the custom files all to a new folder so user won't see conflict.

Reply:
Makes sense. Is it worth adding a section to fork source repo and make changes locally, maybe before Setup GKE ?

Reply:
Done